docs/sources/reference/commandline/cli.md

```diff
@@ -1716,6 +1716,9 @@ You can remove an image using its short or long ID, its tag, or its digest. If
 an image has one or more tag or digest reference, you must remove all of them
 before the image is removed.
 
+If the -f flag is used and the image's short or long ID is specified, the
```

Comment:
Some constructions are awkward.  This gist contains fix:

https://gist.github.com/6cbfbf7b730ef42ed2c7.git